storage/innobase/row/row0mysql.cc

```diff
@@ -1839,7 +1865,9 @@ class ib_dec_counter {
 @param[in,out]	prebuilt	prebuilt struct in MySQL handle
 @return error code or DB_SUCCESS */
 dberr_t
-row_update_for_mysql(row_prebuilt_t* prebuilt)
+row_update_for_mysql_using_upd_graph(
+	row_prebuilt_t*	prebuilt,
+	bool		vers_set_fields)
```

Comment:
Is there a reason to rename this function?

Reply:
I guess this was lost on merge (earlier version had such name).